tensorflow/compiler/xrt/kernels/xrt_execute_op.cc

```diff
@@ -567,11 +571,13 @@ Status XRTExecuteChainedOp::DoWork(OpKernelContext* context) {
 
     return ExecuteComputation(
         context, memory_manager.get(), &device_ref, executable, input_tuples,
-        /*release_inputs=*/false, stream, rng_seed, config.common_config());
+        /*release_inputs=*/false, stream, rng_seed, config.common_config(),
+        device_ref.GetMemoryAllocator());
```

Comment:
ditto, we don't really need to pass `allocator` here since we also pass `device_ref`

Reply:
Done.